docs/migration/migrate_8_0.asciidoc

```diff
@@ -50,41 +50,41 @@ for example, `logstash-*`.
 
 [float]
 ==== Default logging timezone is now the system's timezone
-*Details:* In prior releases the timezone used in logs defaulted to UTC.  We now use the host machine's timezone by default.
+*Details:* In prior releases the timezone used in logs defaulted to UTC. We now use the host machine's timezone by default.
 
 *Impact:* To restore the previous behavior, in kibana.yml use the pattern layout, with a date modifier:
 [source,yaml]
 -------------------
 logging:
   appenders:
     console:
-      kind: console
+      type: console
       layout:
-        kind: pattern
+        type: pattern
         pattern: "%date{ISO8601_TZ}{UTC}"
 -------------------
 See https://github.com/elastic/kibana/pull/90368 for more details.
 
 [float]
 ==== Responses are never logged by default
-*Details:* Previously responses would be logged if either `logging.json` was true, `logging.dest` was specified, or a `TTY` was detected.
+*Details:* Previously responses would be logged if either `logging.json` was true, `logging.dest` was specified, or a `TTY` was detected. With the new logging configuration, these are provided by a dedicated logger.
```

Comment:
> With the new logging configuration,

Should we provide a link to the logging service config? Ideally, in all the places when we refer to it.

Reply:
That is planned for the follow up PR. We can't seem to limbo to docs we're adding from the same PR.